template/src/main.js

```diff
@@ -585,6 +585,13 @@ function init () {
     if ($(id).length > 0) { $('html,body').animate({ scrollTop: parseInt($(id).offset().top) }, 0); }
   }
 
+  /**
+   * Off-Canvas side toggle navigation
+   */
+  $('[data-toggle="offcanvas"]').click(function () {
```

Comment:
new code should not use jquery, but vanilla js

Reply:
Click event listener is now vanilla.